x-pack/plugins/features/server/oss_features.ts

```diff
@@ -28,7 +28,7 @@ export const buildOSSFeatures = ({ savedObjectTypes, includeTimelion }: BuildOSS
           app: ['discover', 'kibana'],
           catalogue: ['discover'],
           savedObject: {
-            all: ['search', 'query', 'index-pattern'],
+            all: ['search', 'query', 'index-pattern', 'background-session'],
```

Comment:
Since we want the sub-feature to automatically be included in this `all` privilege, we should remove it here and rely on the sub-feature privilege to grant this when necessary.

```suggestion
            all: ['search', 'query', 'index-pattern'],
```